$NetBSD: patch-generate_unix_iasl_Makefile,v 1.1 2024/08/04 10:52:06 nia Exp $

"We can't use our normal warning flags because the files aren't
necessarily even ANSI C - let's use Wall and Werror instead".
Interesting logic... let's not make the errors fatal.

--- generate/unix/iasl/Makefile.orig	2024-08-04 10:49:24.308561573 +0000
+++ generate/unix/iasl/Makefile
@@ -320,19 +320,19 @@ $(OBJDIR)/prparserparse.c $(OBJDIR)/prpa
 # by the utilities above and they are not necessarily ANSI C, etc.
 #
 $(OBJDIR)/aslcompilerlex.o :   $(OBJDIR)/aslcompilerlex.c
-	$(CC) -c $(CFLAGS) -Wall -Werror -o$@ $<
+	$(CC) -c $(CFLAGS) -Wall -o$@ $<
 
 $(OBJDIR)/aslcompilerparse.o : $(OBJDIR)/aslcompilerparse.c
-	$(CC) -c $(CFLAGS) -Wall -Werror -o$@ $<
+	$(CC) -c $(CFLAGS) -Wall -o$@ $<
 
 $(OBJDIR)/dtparserlex.o :      $(OBJDIR)/dtparserlex.c
-	$(CC) -c $(CFLAGS) -Wall -Werror -o$@ $<
+	$(CC) -c $(CFLAGS) -Wall -o$@ $<
 
 $(OBJDIR)/dtparserparse.o :    $(OBJDIR)/dtparserparse.c
-	$(CC) -c $(CFLAGS) -Wall -Werror -o$@ $<
+	$(CC) -c $(CFLAGS) -Wall -o$@ $<
 
 $(OBJDIR)/prparserlex.o :      $(OBJDIR)/prparserlex.c
-	$(CC) -c $(CFLAGS) -Wall -Werror -o$@ $<
+	$(CC) -c $(CFLAGS) -Wall -o$@ $<
 
 $(OBJDIR)/prparserparse.o :    $(OBJDIR)/prparserparse.c
-	$(CC) -c $(CFLAGS) -Wall -Werror -o$@ $<
+	$(CC) -c $(CFLAGS) -Wall -o$@ $<
